cache/entries.go

```diff
@@ -44,6 +47,23 @@ func (e *PasswdEntry) args() []interface{} {
 	}
 }
 
+// Column returns the information from the requested columns or an
+// empty string if no column is known.
+func (e *PasswdEntry) Column(col int) string {
+	switch col {
+	case 0:
+		return e.Name
+	case 2:
+		// Spooky cast notice!  This is a blind cast from a
+		// uint32 to an int, which should generally be safe,
+		// but could lead to a really hard to troubleshoot bug
+		// later.
+		return strconv.Itoa(int(e.UID))
```

Comment:
You could use `fmt.Sprintf("%d", e.UID)` if you want to avoid the cast.
But I'm fine with the current solution.

Reply:
This was the only thing causing an import of strconv, so I removed it in favor of the Sprintf solution.